tests/utils/test_settings_utils.py

```diff
@@ -0,0 +1,164 @@
+import json
+import os
+from pathlib import Path
+from unittest import mock
+from django.test import TestCase, override_settings
+from django.conf import settings
+from arches.settings_utils import generate_frontend_configuration
+
+
+# these tests can be run from the command line via
+# python manage.py test tests.utils.test_settings_utils --settings="tests.test_settings"
+
+
+class TestFrontendConfigurationGeneration(TestCase):
```

Comment:
Thanks for adding this. Now that we have a test in place, let's keep ourselves accountable by no longer excluding it from the coverage calculation. Would you mind adjusting the `settings*.py` to `*settings.py` or similar?

Reply:
@jacobtylerwalls - I'm slightly confused by this. When I submitted my initial change, the coverage check failed the PR despite me only changing the `settings_utils.py` and `release/7.6.x.md` files. The above note suggests that `settings_utils.py` would have been excluded? This is the only reason I added the test!!

Reply:
... also, if I include `settings_utils.py` in the coverage at this point, am I going to have to write tests for the other functions in the module for this to be accepted?

Reply:
Oh I certainly wouldn't ask you to volunteer for testing a bunch of unrelated stuff, no worries.

>  the coverage check failed the PR despite me only changing the settings_utils.py

I can't speak to why that comparison failed, but I wonder if there was a slight drift between your branch and upstream dev/7.6.x

If you go to the actions tab and look at the coverage reports generated, you'll see there's no arches/settings_utils.py being measured

Reply:
The coverage check has to be taken with a grain of salt because of the "falling denominator" problem -- it's always acceptable to remove tested code.

Reply:
thanks for reviewing and sorting out @jacobtylerwalls.